neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDriver.java

```diff
@@ -60,6 +62,7 @@
  * this class directly, it registers automatically with the {@link DriverManager}.
  *
  * @author Michael J. Simons
+ * @author Rouven Bauer
```

Comment:
👍 

I do like that (the general approach and the addition), but opinions here vary.